File: StatusBarSpeedometer/app/src/main/java/ch/rmy/android/statusbar_tacho/units/FeetPerSecondUnit.java
package ch.rmy.android.statusbar_tacho.units;

import ch.rmy.android.statusbar_tacho.R;

public class FeetPerSecondUnit extends Unit {

    @Override
    public int getNameRes() {
        return R.string.unit_fts;
    }

    @Override
    public float convertSpeed(float metersPerSecond) {
        return metersPerSecond * 3.28084f;
    }
}

File: StatusBarSpeedometer/app/src/main/java/ch/rmy/android/statusbar_tacho/units/FeetPerSecondUnit.kt
package ch.rmy.android.statusbar_tacho.units

import ch.rmy.android.statusbar_tacho.R

class FeetPerSecondUnit : Unit() {

    override val nameRes: Int
        get() = R.string.unit_fts

    override val maxValue: Int
        get() = 160

    override val steps: Int
        get() = 8

    override fun convertSpeed(metersPerSecond: Float): Float {
        return metersPerSecond * 3.28084f
    }

}
